spark/src/main/scala/org/apache/mahout/drivers/RowSimilarityDriver.scala

```diff
@@ -135,7 +133,7 @@ object RowSimilarityDriver extends MahoutSparkDriver {
   }
 
   override def process: Unit = {
-    start()
+    start
```

Comment:
no-parenthesis is a proprety style in Scala. E.g. all java getters by default are styled without (). I am guessing start() is really a routine here, not a property. The convention is that a thing without () must not change object state or have any other side effects.  The setters (optionally) then are encoded using `def start_=(value)` form.